shoop/front/middleware.py

```diff
@@ -126,8 +126,12 @@ def process_view(self, request, view_func, *view_args, **view_kwargs):
             return maintenance_response
 
     def _get_maintenance_response(self, request, view_func):
-        if "login" in view_func.__name__ or getattr(view_func, "maintenance_mode_exempt", False):
-            return
+        if getattr(view_func, "maintenance_mode_exempt", False):
+            return None
+        if "login" in view_func.__name__:
+            return None
+        if getattr(request, "resolver_match") and getattr(request.resolver_match, "app_name") == "shoop_admin":
```

Comment:
Maybe make this

```python
resolver_match = getattr(request, "resolver_match", None)
if resolver_match and resolver_match.app_name == "shoop_admin":
```

We can probably assume that if we have a `resolver_match`, it will have `app_name`.